src/rlookup.c

```diff
@@ -670,7 +670,9 @@ static int getKeyCommonJSON(const RLookupKey *kk, RLookupRow *dst, RLookupLoadOp
   char *keyPtr = options->dmd ? options->dmd->keyPtr : (char *)options->keyPtr;
   if (!*keyobj) {
 
-    *keyobj = japi->openKeyFromStr(ctx, keyPtr);
+    RedisModuleString* keyName = RedisModule_CreateString(ctx, keyPtr, strlen(keyPtr));
+    *keyobj = japi->openKey_withFlags(ctx, keyName, REDISMODULE_OPEN_KEY_NOEFFECTS);
```

Comment:
Check that this API is available (using `japi->ver`)
Or in module init - check for V5 API and if not available - set `japi` to null